New footprint added (Schurter FAB fuse holder for 5x20mm cylinder fuses)

tk5ep opened the pull request:
Fuseholder_Cylinder-5x20mm_Schurter_FAB_0031-3553_Horizontal_Closed.kicad_mod

poeschlr commented:
Naming discussion and 3d model pull request: https://github.com/KiCad/kicad-packages3D/pull/142

Datasheet: https://www.schurter.com/bundles/snceschurter/epim/_ProdPool_/newDS/en/typ_FAB.pdf

It seems the order number 0031.3553 does not exist. (can not find it on the schurter webside.)
I can only find two order numbers for the FAB fuse holder. (0031.3551 and 0031.3558) I think the straight and kinked option refers to the leads. (kinked pins have a kink as shown in the datasheet, the straight pins are similar to your 3d model.)

I don't think the two versions require a different footprint i would therefore mask the last digit of the order number via a wildcard. This would result in the following name:
`Fuseholder_Cylinder-5x20mm_Schurter_FAB_0031-355x_Horizontal_Closed`

The following things need to be corrected to comply to the KLC:
Graphical stuff:
- The silk screen should be outside of the body (outside the fab outline)
- Courtyard is missing (
- Keep the reference text as Ref** (such that it is the same as for other footprints.)
- Add a second reference on the fab layer. (Add a user text field with text set equal to %R -> will be replaced by the reference)
- The footprint is offset by 0.1mm to the right. (pin 1 should be on 0,0)
- Line width for the fab layer lines should be 0.1mm
- Line width for the silk layer lines should be 0.12mm
- Remove the silk screen from the pads (tick in the pad options)
- Do not overlap silk lines with pads. (Should have a clearance of 0.2mm to copper.)

Footprint properties:
- 3d settings: these should be scaling 1, offset and rotation 0 (for x,y and z settings. In other words: Align your 3d model to the footprint instead of using this legacy feature)
- Footprint attributes: set to normal+insert (THT)
- Do not set Pad and Soldermask clearance in the footprint options. (This depends on the application not the part used.)
- Add a link to the datsheet in the description field.


I think it is faster for you if you take my footprint. (I have corrected a few things compared to the one i attached to the 3d model pull request.) At least you can use it as a guide if you are unsure what my requests mean.
[Fuseholder_Cylinder-5x20mm_Schurter_FAB_0031-355x_Horizontal_Closed.kicad_mod.txt](https://github.com/KiCad/Fuse_Holders_and_Fuses.pretty/files/1391306/Fuseholder_Cylinder-5x20mm_Schurter_FAB_0031-355x_Horizontal_Closed.kicad_mod.txt)

poeschlr commented:
Some measurements of your footprint are not correct.

![screenshot from 2017-10-17 16-50-30](https://user-images.githubusercontent.com/18327350/31672240-4ec49496-b35c-11e7-9636-cc475e622eba.png)

tk5ep commented:
Hello René,

Well, i think i'm giving you more work than help !!

You would work faster than correcting me all time... Anyway thanks for all
these infos. I will take them as a guideline for future work. (not sure if
i will make other pushes... hi hi  )

Yes, you're right the 0031-3553 seems to be a typo found on several
reseller sites (Mouser, etc) It's definitively 3551 or 3558.

I suggest that you push your own footprint and that we leave mine in the
garbage...

I did redraw the 3D shape yesterday evening and modified some things, the
actual model should be OK.

I attach it in FC format.

On your other mail where you inform me about some dimension errors, i'm not
sure if the front part is 10.6 mm . The datasheet says 10.06, but the
center is given at 5mm. So i left it at 5mm diameter. I think it's within
tolerances and should solve this incoherence.

I hope this helps...

Patrick

2017-10-17 16:22 GMT+02:00 Rene Pöschl <notifications@github.com>:

> Naming discussion and 3d model pull request: KiCad/kicad-packages3D#142
> <https://github.com/KiCad/kicad-packages3D/pull/142>
>
> Datasheet: https://www.schurter.com/bundles/snceschurter/epim/_
> ProdPool_/newDS/en/typ_FAB.pdf
>
> It seems the order number 0031.3553 does not exist. (can not find it on
> the schurter webside.)
> I can only find two order numbers for the FAB fuse holder. (0031.3551 and
> 0031.3558) I think the straight and kinked option refers to the leads.
> (kinked pins have a kink as shown in the datasheet, the straight pins are
> similar to your 3d model.)
>
> I don't think the two versions require a different footprint i would
> therefore mask the last digit of the order number via a wildcard. This
> would result in the following name:
> Fuseholder_Cylinder-5x20mm_Schurter_FAB_0031-355x_Horizontal_Closed
>
> The following things need to be corrected to comply to the KLC:
> Graphical stuff:
>
>    - The silk screen should be outside of the body (outside the fab
>    outline)
>    - Courtyard is missing (
>    - Keep the reference text as Ref** (such that it is the same as for
>    other footprints.)
>    - Add a second reference on the fab layer. (Add a user text field with
>    text set equal to %R -> will be replaced by the reference)
>    - The footprint is offset by 0.1mm to the right. (pin 1 should be on
>    0,0)
>    - Line width for the fab layer lines should be 0.1mm
>    - Line width for the silk layer lines should be 0.12mm
>    - Remove the silk screen from the pads (tick in the pad options)
>    - Do not overlap silk lines with pads. (Should have a clearance of
>    0.2mm to copper.)
>
> Footprint properties:
>
>    - 3d settings: these should be scaling 1, offset and rotation 0 (for
>    x,y and z settings. In other words: Align your 3d model to the footprint
>    instead of using this legacy feature)
>    - Footprint attributes: set to normal+insert (THT)
>    - Do not set Pad and Soldermask clearance in the footprint options.
>    (This depends on the application not the part used.)
>    - Add a link to the datsheet in the description field.
>
> I think it is faster for you if you take my footprint. (I have corrected a
> few things compared to the one i attached to the 3d model pull request.) At
> least you can use it as a guide if you are unsure what my requests mean.
> Fuseholder_Cylinder-5x20mm_Schurter_FAB_0031-355x_
> Horizontal_Closed.kicad_mod.txt
> <https://github.com/KiCad/Fuse_Holders_and_Fuses.pretty/files/1391306/Fuseholder_Cylinder-5x20mm_Schurter_FAB_0031-355x_Horizontal_Closed.kicad_mod.txt>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/KiCad/Fuse_Holders_and_Fuses.pretty/pull/17#issuecomment-337247516>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABlBhURHGPZHbcINpIq4CQNAzIGhlKmXks5stLgXgaJpZM4P7lsS>
> .
>



-- 
Patrick Egloff
email : pegloff@gmail.com
Web page : http://www.egloff.eu

SchrodingersGat commented:
Closing as the replacement PR has been merged.
